### PR TITLE
added two test strings to en.yml file

### PR DESCRIPTION
**Pull request by ErinClaudio**
### What github issue is this PR for, if any?
WIP #1257

### What changed, and why?
I added two text strings and commented in the en.yml file. I want to make sure I am using the proper nesting conventions before continuing. 

### How will this affect user permissions?
N/A

### How is this tested? (please write tests!) 💖💪
N/A

### Screenshots please :)
Screen Shot 2021-01-14 at 3 04 57 PM

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -57,3 +57,11 @@ en:
       full: "%B %-d, %Y"
       youth_date_of_birth: "%B %Y"
       short_date: "%-m/%d"
+  page:
+    #new: I am not sure if the file name should be add?
+      judge: #Should this be Plural as the directory name is plural. Should the
+        title: "New Judge"
+  page:
+    judge:
+      #edit:
```
Comment (compwron) [inline config/locales/en.yml:66]: leave out `edit`
indentation matters :)

**Comment by compwron** [comment]
this is looking promising! :)

File: config/locales/en.yml [hunk 2]
```diff
@@ -57,3 +57,11 @@ en:
...
+        title: "Edit Judge
```
Comment (littleforest) [inline config/locales/en.yml]: @ErinClaudio thanks for working on this! I would start with defining the active record model and attributes for Judge according to [this documentation](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models). For example:

```
en:
  activerecord:
    models:
      judge:
        one: Judge
        other: Judges
    attributes:
      judge:
        name: Name
```


Once that is in place, you could create entries for new, edit, delete, etc. like so:

```
en:
  new_record: "New %{model}"
  edit_record: "Edit %{model}"
  destroy_record: "Delete %{model}"
```

And then call it like so:

```
@judge = Judge.first
model_name = @judge.class.model_name.human
new_judge_str = I18n.t(:new_record, model: model_name)
```

Comment (ErinClaudio) [reply config/locales/en.yml]: Thank you for the documentation!

**Comment by compwron** [comment]
This is looking promising! The build seems upset and I'm not sure why... I will try to take a look later today but feel free to beat me to it

**Comment by ErinClaudio** [comment]
> This is looking promising! The build seems upset and I'm not sure why... I will try to take a look later today but feel free to beat me to it.

- one error- I missed a closing quote. 
- and I believe I cannot nest with repeating keys as I did below
`page:
    judge:
      title: "New Judge"
  page:
    judge:
      title: "Edit Judge"
      `

**Comment by ErinClaudio** [comment]
Hello,
I have attempted to clean this up.

- lines 52 - 55

		I added `judge: new_title: "New Judge"` and `judge: edit_title: "Edit Judge."` Thinking that those key values should be under `page:` (line 48)

- line 73 and 75

		I added create and update keys.

- line 77 - 82 

I changed the keys to (79)`new_title:` and (82) `edit_title: `. The original repeating key might have been causing a failure. Please let me know which of the commented out lines I should remove. I look forward to your feedback and improvement.

File: config/locales/en.yml [hunk 3]
```diff
@@ -57,3 +61,22 @@ en:
       full: "%B %-d, %Y"
       youth_date_of_birth: "%B %Y"
       short_date: "%-m/%d"
+  activerecord:
+    models:
+      judge:
+        one: Judge
+        other: Judges
```
Comment (compwron) [inline config/locales/en.yml:68]: Let's leave out `other:` and use https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize in the code instead

File: config/locales/en.yml [hunk 4]
```diff
@@ -57,3 +61,22 @@ en:
...
+        name: Name
+  new_record: "New %{model}"
+ #create_record: "Create %{model}"
+  edit_record: "Edit %{model}"
+ #update_record: "Update %{model}"
+  destroy_record: "Delete %{model}"
+  page:
+    judge:
+      new_title: "New Judge"
+  page:
```
Comment (compwron) [inline config/locales/en.yml:80]: YAML doesn't allow repeated keys like this- see https://stackoverflow.com/questions/47668308/duplicate-key-in-yaml-configuaration-file for an example

**Comment by compwron** [comment]
Ping me in slack if you want to pair on this- I'm around a lot today and I can make time this week :)

File: config/locales/en.yml (hunk 4 above)
Comment (scottolsen) [reply config/locales/en.yml:80]: For views you can also do something like
```yml
# config/locals/en.yml
judges:
  new:
    title: 'New Judge'
  edit:
    title: 'Edit Judge'
```

And in the view you can just reference it like below.
```ruby
# app/views/judges/edit.html.erb
<%= render partial: "form", locals: {title: t('.title'), judge: @judge} %>

# app/views/judges/new.html.erb
<%= render partial: "form", locals: {title: t('.title'), judge: @judge} %>
```

More info [here](https://guides.rubyonrails.org/i18n.html#lazy-lookup).

**Review by compwron — APPROVED** [review]

**Comment by compwron** [comment]
Heck yes!!

**Pull request merged**